project/Build.scala

```diff
@@ -207,11 +207,28 @@ object DottyBuild extends Build {
     ).
     settings(publishing)
 
+  // until sbt/sbt#2402 is fixed (https://github.com/sbt/sbt/issues/2402)
+  lazy val cleanBridge = TaskKey[Unit]("clean-sbt-bridge", "delete dotty-sbt-bridge cache")
```

Comment:
the convention is to use `a-b-c`
http://www.scala-sbt.org/0.13/sxr/sbt/Keys.scala.html#sbt.Keys.logLevel